bundles/org.eclipse.ui/META-INF/MANIFEST.MF

```diff
@@ -9,9 +9,7 @@ Bundle-Vendor: %Plugin.providerName
 Bundle-Localization: plugin
 Export-Package: org.eclipse.ui.internal;x-internal:=true
 Require-Bundle: org.eclipse.core.runtime;bundle-version="[3.29.0,4.0.0)",
- org.eclipse.swt;bundle-version="[3.124.0,4.0.0)";visibility:=reexport,
```

Comment:
Reexports can not be removed even if they are unused

Reply:
Thanks for the review. reverted the `SWT` entry that has the reexport visibility. please have a look.
btw I was using the `PDE` tool to clean up the dependencies. Do you think the tool needs to care about the visibility attribute?

Reply:
It should have an option to keep such entries, in general one *can* remove, but for eclipse platform it would be an API break, so if you can enhance/improve it it would be great.